src/i18n.js

```diff
@@ -26,15 +23,18 @@ function loadLocaleConfig() {
   return messages;
 }
 
-const i18n = new VueI18n({
+const i18n = createI18n({
+  legacy: true,
+  globalInjection: true,
   locale: CONFIG.locale,
   fallbackLocale: CONFIG.fallbackLocale,
   messages: loadLocaleConfig(),
-  // Todo: Workaround for https://github.com/kazupon/vue-i18n/issues/563
+  // Workaround for https://github.com/kazupon/vue-i18n/issues/563
   postTranslation: (value, path) => {
     if (value === "") {
       const parts = path.split('.');
-      let message = i18n.messages[CONFIG.fallbackLocale];
+      // Access messages in a mode-agnostic way
+      let message = i18n.global.getLocaleMessage(CONFIG.fallbackLocale);
```

Comment:
This needs good testing, it's an edge-cases that not happend very often, but e.g. in Japanese.